### DISCUSSION TITLE
any news on upscaler not affecting the UI?

### DISCUSSION THREAD

**Discussion by rpinheiro0** [praydog/REFramework#898] [General Questions] [answered]
Any sort of beta version of it maybe? 

It would be really good for RE4R because the ghosting on the UI can be very distracting, especially for the quick slots.

**Comment by praydog** [comment, marked as answer]
I have gotten this working recently in SF6 and RE4 privately but it has some issues like memory leaks and the UI not being the correct color. Working through this requires research and trial and error which can be quite boring, so I don't work on it constantly.

**Reply by rpinheiro0** [reply]
That makes perfect sense. Working on trial and error is exhausting. Thanks for the update!

**Comment by dbox2power** [comment]
Hi Praydog, I just reinstalled my PC and now I wanted to add your great upscaler REFramework to my RE Games, but found out that your downloads on Github got expired. Could you please reupload the REFrameworks with upscaler for the RE games again? :) Can´t wait to play with DLSS again. Many thanks for your great work! 

EDIT: just found your new Download Page. THX! 
https://github.com/praydog/REFramework/actions/runs/6085045392
